lms/envs/production.py

```diff
@@ -741,6 +741,9 @@
 GOOGLE_ANALYTICS_LINKEDIN = AUTH_TOKENS.get('GOOGLE_ANALYTICS_LINKEDIN')
 GOOGLE_SITE_VERIFICATION_ID = ENV_TOKENS.get('GOOGLE_SITE_VERIFICATION_ID')
 
+##### GOOGLE TAG MANAGER IDS #####
+GOOGLE_TAG_MANAGER_ACCOUNT = AUTH_TOKENS.get('GOOGLE_TAG_MANAGER_ACCOUNT')
```

Comment:
This value is got from the lms.auth.json file that is generated by ansible in each deployment, we can add that config to the **EDXAPP_AUTH_EXTRA** ansible variable and the lms.auth.json will be generated with that tag.